src/components/content/order/types/Regions.ts

```diff
@@ -6,7 +6,7 @@
 /**
  * The regions inside Area
  */
-export interface Region {
+export interface Regions {
```

Comment:
this is still a single `Region` right? or does this type store multiple regions always?